### DISCUSSION TITLE
Homebrew uninstall Mac M1 Terminal Error

### DISCUSSION THREAD

**Discussion by MikeKasp** [Homebrew/discussions#2812] [Getting started]
Hi there, 

after the uninstallation of Homebrew with the uninstall code from the Homebrew git-repo my terminal shows me the following message: "/Users/myusername/.zprofile:7: no such file or directory: /opt/homebrew/bin/brew"

Does anyone know how to delete this error? I have read something on google about typing "nano .zprofile " in the terminal and then comment the line 'eval "$(/opt/homebrew/bin/brew shellenv)' out. But I'am not sure and don't want to harm my system.

Thanks in advance
Mike

**Comment by carlocab** [comment]
> I have read something on google about typing "nano .zprofile " in the terminal and then comment the line 'eval "$(/opt/homebrew/bin/brew shellenv)' out.

You can do this. You won't harm your system. It may be easier to do this using TextEdit by doing
```
open ~/.zprofile
```
in your terminal. This will open TextEdit. Delete the line that says `eval "$(/opt/homebrew/bin/brew shellenv)` and save the file. If the file is empty after you deleted the line, you can do
```
rm ~/.zprofile
```
(also in your terminal) to delete it. You don't need it anymore (but only if it's empty!).

**Reply by MikeKasp** [reply]
Thanks, that was easy :-)

**Reply by MikeKasp** [reply]
Now my pip is gone. If I type "pip list" I get the following message "zsh: command not found: pip"

**Reply by carlocab** [reply]
How did you install `pip` before?

**Reply by MikeKasp** [reply]
Comes it with the python installer? 

If I type pip3 --version this comes. 

usernam@MacBook-Pro-von-user ~ % pip3 --version
pip 21.3.1 from /Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/pip (python 3.10)

**Reply by carlocab** [reply]
You'll have to check with Python.org then (assuming that's where you got your Python installer from), as this seems unrelated to Homebrew and I'm not familiar with what their installer does or doesn't install.

**Reply by MikeKasp** [reply]
With pip3 list I got the list.

**Reply by MikeKasp** [reply]
Before I have installed Homebrew, "pip list" does work. After installing and deleting Homebrew this does't work anymore. Brew does not effect pip?

**Reply by carlocab** [reply]
It does not.

**Reply by MikeKasp** [reply]
Ok thanks.

**Reply by chandz102** [reply]
this did the trick! thanks!
